x-pack/plugins/security_solution/public/detections/pages/detection_engine/rules/all/actions.ts

```diff
@@ -141,7 +211,7 @@ async function defaultSuccessHandler(
   switch (action) {
     case BulkAction.export:
       title = i18n.RULES_BULK_EXPORT_SUCCESS;
-      text = i18n.RULES_BULK_EXPORT_SUCCESS_DESCRIPTION(summary.succeeded, summary.total);
+      text = getExportSuccessToastMessage(summary.succeeded, summary.total);
```

Comment:
When I tried to add it here, got TS error

```
(property) title?: string | MountPoint<HTMLElement> | undefined
Variable 'title' is used before being assigned.ts(2454)
```

Same error can be received, if  you remove one of enum values(without using `assertUnreachable`). So, I guess, it's protected here from non exhaustive switch in that case and no need to introduce it here. Otherwise , we would need to initialize  title variable with some mock value to get over TS checks

Reply:
The exact code function body I suggested doesn't lead to TS error you mentioned on my machine, did you try exactly this one?

Reply:
No, title and text were still `undefined`. Which makes current code as I mentioned before safe from non exhaustive switch.
But I agree, let's try to use `assertUnreachable` where is possible in future